### PR TITLE
Bug 2304905: [release-4.16] Add cluster name and id on clients' list page

### PR DESCRIPTION
**Pull request by SanjalKatiyar**
Screenshot 2024-08-22 at 6 22 50 PM

### CODE REVIEW

**Review by GowthamShanmugam — COMMENTED** [review]
LGTM

File: packages/odf/components/storage-consumers/client-list.tsx [hunk 1]
```diff
@@ -268,7 +268,9 @@ const StorageClientRow: React.FC<
         {obj?.status?.client?.name || '-'}
       </TableData>
       <TableData {...tableColumns[1]} activeColumnIDs={activeColumnIDs}>
-        {obj?.status?.client?.clusterId || '-'}
+        {`${obj?.status?.client?.clusterName || '-'} (${
+          obj?.status?.client?.clusterId || '-'
```
Comment (bipuladh) [inline packages/odf/components/storage-consumers/client-list.tsx:272]: Why are we falling back to clusterId?

Comment (SanjalKatiyar) [reply packages/odf/components/storage-consumers/client-list.tsx:272]: We are not falling back, we are displaying both together: `ClusterName (ClusterId)`.

It's also same as what's happening in the master.

**Pull request merged**